.travis.yml

```diff
@@ -19,10 +26,6 @@ before_install:
  # Useful for debugging any issues with conda
  - conda info -a
 
- # create and activate the build/test environment
- - conda create -q -n tenv python=$TRAVIS_PYTHON_VERSION pip cmake
```

Comment:
Yes, the creation and activation of a dedicated conda environment is quite optional, using the root environment should be fine, it may require fetching a miniconda distribution that matches the intended major/minor version of python (3.4 and 3.5 I'm afraid will require different packages), but this is not probably going to add any significant complexity.